src/org/opensolaris/opengrok/configuration/messages/ConfigMessage.java

```diff
@@ -17,7 +17,7 @@
  * CDDL HEADER END
  */
 
- /*
+/*
  * Copyright (c) 2017, 2017, Oracle and/or its affiliates. All rights reserved.
```

Comment:
copyright

Reply:
fixed, hopefully we won't have to deal with this anymore soon.